integration/tests/cook/test_cli.py

```diff
@@ -623,6 +627,10 @@ def test_ssh_job_uuid(self):
         self.assertIn(f'-t {hostname} cd', stdout)
         self.assertIn('; bash', stdout)
 
+    @unittest.skipIf(util.has_ephemeral_hosts(), 'If the cluster under test has ephemeral hosts, then it is generally '
+                                                 'a bad idea to use HOSTNAME EQUALS constraints, because it can cause '
+                                                 'the process responsible for launching hosts to launch hosts that '
+                                                 'never get used')
```

Comment:
Can we have this string in a constant somewhere and refer to the constant in the annotation?

Reply:
Done